sonic_installer/main.py

```diff
@@ -147,6 +147,20 @@ def remove_image(image):
         run_command('grub-set-default --boot-directory=' + HOST_PATH + ' 0')
         click.echo('Image removed')
 
+# TODO: Embed tag name info into docker image meta data at build time,
+# and extract tag name from docker image file.
+def get_docker_tag_name(image):
+    # Try to get tag name from label metadata
+    cmd = "docker inspect  --format '{{.ContainerConfig.Labels.Tag}}' " + image
```

Comment:
Remove extra space between `inspect` and  `--format`

Reply:
done.